bc_obps/service/operation_service.py

```diff
@@ -1,5 +1,7 @@
 from uuid import UUID
 from django.db import transaction
+from registration.emails import send_boro_id_application_email
```

Comment:
oof - sorry you had to create the operation service (I know that was supposed to be done in another ticket). I was trying to avoid doing this when I was working on it, but it was extremely tricky to do when the operator service was already in use.